vps/api.go

```diff
@@ -596,3 +596,16 @@ func GetAvailableAvailabilityZones(c gotransip.Client) ([]AvailabilityZone, erro
 
 	return v.V, err
 }
+
+// ConvertVpsBackupToSnapshot converts a backup to a separate snapshot for a VPS
+func ConvertVpsBackupToSnapshot(c gotransip.Client, vpsName, description string, vpsBackupID int64) error {
+	sr := gotransip.SoapRequest{
+		Service: serviceName,
+		Method:  "convertVpsBackupToSnapshot",
+	}
+	sr.AddArgument("vpsName", vpsName)
+	sr.AddArgument("description", description)
+	sr.AddArgument("vpsBackupID", fmt.Sprintf("%d", vpsBackupID))
```

Comment:
not sure if it should be ID or Id. Or if it matters at all

Reply:
We're already using `vpsBackupID` in `RevertVpsBackup` (https://github.com/transip/gotransip/blob/master/vps/api.go#L397) so this keeps it consistent. Also, golint will complain about it: `func parameter vpsBackupId should be vpsBackupID (golint)`

Reply:
The string argument to `AddArgument` is with Id though:
```
sr.AddArgument("vpsBackupId", fmt.Sprintf("%d", vpsBackupID))
```

Reply:
true but one is the go variable and the other is the variable it represents in the SOAP request

Reply:
You're completely right, that's bad copy/pasting probably.